pkg/client/client.go

```diff
@@ -44,6 +46,8 @@ const (
 	AwsCredsSecretIDKey = "aws_access_key_id"
 	// AwsCredsSecretAccessKey is secret key containing AWS Secret Key
 	AwsCredsSecretAccessKey = "aws_secret_access_key"
+
+	globalInfrastuctureName = "cluster"
```

Comment:
can you reference where this magic "cluster" is coming from so others have context? i.e the line of code in the installer where this is instantiated

Reply:
👍

Reply:
https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L10